app/views/devise/registrations/_password_details.html.erb

```diff
@@ -1,44 +1,55 @@
 
-    <%= hidden_field_tag :skip_personal_details, "true" %>
+  <p class="form-control-static"><%= _('If you would like to change your password please complete the following fields.') %></p>
 
-    <div class="form-input">
-      <%= f.label(:current_password, _('Current password'), class: 'required') %>
-      <%= f.password_field(:current_password, class: 'left-indent required input-medium') %>
-      <span role="" id="current_password_error"
-            class="error-tooltip-right left-indent"></span>
+  <%= hidden_field_tag :skip_personal_details, "true" %>
+
+  <div class="form-group">
+    <%= f.label(:current_password, _('Current password'), class: 'col-sm-2 control-label') %>
+    <div class="col-sm-6">
+      <%= f.password_field(:current_password,  class: 'form-control', "aria-required": true ) %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label(:password, _('New password'),  class: 'col-sm-2 control-label') %>
+    <div class="col-sm-6"> 
+      <%= f.password_field(:password,  class: 'form-control', "aria-required": true , id: 'user_new_password') %>
     </div>
+  </div>
 
-    <div class="form-input">
-      <%= f.label(:password, _('New password'), class: 'required') %>
-      <%= f.password_field(:password, class: 'left-indent required input-medium', id: 'user_new_password') %>
-      <span role="" id="password_error"
-            class="error-tooltip-right left-indent"></span>
+  <div class="form-group">
+    <%= f.label(:current_password, _('Current password'), class: 'col-sm-2 control-label') %>
+    <div class="col-sm-6">
+      <%= f.password_field(:current_password,  class: 'form-control', "aria-required": true ) %>
     </div>
+  </div>
 
-    <div class="form-input">
-      <%= f.label(:password_confirmation, _('Password confirmation'), class: 'required') %>
-      <%= f.password_field(:password_confirmation, class: 'left-indent required input-medium') %>
-      <span role="" id="password_confirmation_error"
-            class="error-tooltip-right left-indent"></span>
+  <div class="form-group">
+    <%= f.label(:password_confirmation, _('Password confirmation'),  class: 'col-sm-2 control-label') %>
+    <div class="col-sm-6">
+      <%= f.password_field(:password_confirmation, class: 'form-control', "aria-required": true,  ) %>
     </div>
+  </div>
 
-    <div class="form-input">
-      <%= label_tag(:password_show, _('Show passwords')) %>
-      <%= check_box_tag(:password_show, "1", false, class: 'left-indent') %>
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
+      <div class="checkbox">
+        <label>
+           <input type="checkbox" id="passwords_toggle" /><%= _('Show passwords') %>
+        </label>
+      </div>
     </div>
-    
-    <br />
-    <div class='form-input'>
-      <div class="button-spacer">&nbsp;</div>
+  </div>
+
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
       <%= render partial: 'shared/accessible_submit_button',
```

Comment:
We're no longer using the accessible_submit_button, so go ahead and switch this to follow the button on another Bootstrapped page like `<%= f.button(_('Save'), class: "btn btn-default", type: "submit") %>`